Psychtoolbox/PsychDemos/ProceduralColorGratingDemo.m

```diff
@@ -103,18 +113,14 @@ function ProceduralColorGratingDemo(color1,color2,baseColor)
         angle, [], [], baseColor, [], [],...
         [phase, frequency, contrast, sigma]);
 
-    Screen('DrawingFinished', win);
-
     vbl = Screen('Flip', win, vbl + 0.5 * ifi);
     phase = phase - 15;
 end
 
-%--- now we can change the parameters in the shader to be B-Y grating
-% note this takes ~0.7ms 
-glUseProgram(shader);
-glUniform4f(glGetUniformLocation(shader, 'color1'), 0, 0, 1, 1);
-glUniform4f(glGetUniformLocation(shader, 'color2'), 1, 1, 0, 1);
-glUseProgram(0);
+%--- Lets make a Blue/Yellow colour shader 
+Screen('Close',texture);
```

Comment:
Not sure if this Close is needed?

Reply:
It is needed if you don't want to leak textures. At least for showing correct use. sca would close all open textures, but issue warnings if it finds more than 10 open textures.